template/dev

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Can we call this file something different? Like ```enter-container```?

Reply:
Sure, I just wanted to keep it short to not have to remember about name